interview-guide/readme.md

```diff
@@ -55,3 +61,23 @@ Don't be afraid of uncomfortable silences, sometimes you interviewee needs time
 
 ### Reboot an interview
 If an interview is going badly, you can try to reboot the interview by thanking the participant for their time, and engaging in a little small talk. After a while ask your interviewee to answer one last question on the record and slowly reengage the interview. Often people will be more open to your questions after they spoke to you as a person for a bit and it feels as if there is less depending on there answers.
+
+## What to do after the interview.
+
+It is good write down what you thought was the most important things that where discussed, and make notes of why you think these things are important. Do this directly after the interview when the memory is still fresh. This way you have some reference when you are compiling the interview.
```

Comment:
> It is good write down what you thought was the most important things that where discussed and make notes of why you think these things are important.

Sentence does not flow well.

Suggestion: *It is good to write down key points of the discussed subjects and make notes of why you think these takeaways are important.*

> the memory is still fresh.

Suggestion: *your memory is still fresh.*
To make it more personal?